src/mobile/src/main/java/tupperdate/android/home/Home.kt

```diff
@@ -1,98 +1,145 @@
-import androidx.compose.material.Button
+import androidx.compose.foundation.gestures.draggable
+import androidx.compose.foundation.layout.*
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.WithConstraints
+import androidx.compose.ui.gesture.scrollorientationlocking.Orientation
+import androidx.compose.ui.platform.DensityAmbient
+import androidx.compose.ui.platform.LifecycleOwnerAmbient
+import androidx.compose.ui.unit.Dp
 import androidx.compose.ui.unit.dp
 import androidx.ui.tooling.preview.Preview
+import tupperdate.android.appbars.MainBottomBar
+import tupperdate.android.appbars.TitleTopBar
 import tupperdate.android.ui.TupperdateTheme
-import tupperdate.api.AuthenticationApi
+import tupperdate.api.RecipeApi
+import tupperdate.api.api
+import kotlin.math.absoluteValue
 
 @Composable
 fun Home(
-    onButtonClick: () -> Unit,
-    user: AuthenticationApi.Profile?,
+    recipeApi: RecipeApi,
+    onChatClick: () -> Unit,
+    onProfileClick: () -> Unit,
+    onReturnClick: () -> Unit,
+    onRecipeClick: () -> Unit,
     modifier: Modifier = Modifier,
 ) {
-    if (user == null) {
-        HomeDisconnected(
-            onButtonClick,
-            modifier,
-        )
-    } else {
-        HomeConnected(
-            onButtonClick,
-            user,
-            modifier,
-        )
-    }
-}
+    // TODO make a call to front element of recipeApi
+    val presentRecipe = (RecipeApi.Recipe(
+        "Red lobster",
+        "Look at me, am I not yummy ?",
+        "lobster.jpg"
+    ))
 
-@Composable
-private fun HomeDisconnected(
-    onButtonClick: () -> Unit,
-    modifier: Modifier,
-) {
-    Column(
-        modifier.padding(10.dp)
-    ) {
-        Text(text = "Hi there.")
-        Button(onClick = onButtonClick) {
-            Text("Go to onboarding")
+    TitleTopBar(onChatClick = onChatClick, onProfileClick = onProfileClick)
+
+    Row(modifier.fillMaxSize()) {
+        Column(
+            Modifier.fillMaxWidth()
+                .align(Alignment.Bottom)
+                .padding(bottom = veryLittleButtonSize.dp)
+        ) {
+            DisplayRecipeCard(
+                presentRecipe = presentRecipe,
+                onLike = { recipeApi.like(presentRecipe) },
+                onDislike = { recipeApi.dislike(presentRecipe) },
+            )
+            MainBottomBar(
+                onLike = { recipeApi.like(presentRecipe) },
+                onDislike = { recipeApi.like(presentRecipe) },
+                onReturn = onReturnClick,
+                onRecipeClick = onRecipeClick
+            )
         }
     }
 }
 
 @Composable
-private fun HomeConnected(
-    onButtonClick: () -> Unit,
-    user: AuthenticationApi.Profile,
-    modifier: Modifier,
+private fun DisplayRecipeCard(
```

Comment:
The height of the picture doesn't seem to be correct, but we can fix that later. If so, we should add an issue

Reply:
I just tested, and it changes with 0.75 or 0.8 parameter. So it works !